egs/fame/v2/run.sh

```diff
@@ -0,0 +1,402 @@
+
+      steps/make_mfcc.sh --mfcc-config conf/mfcc_16k.conf --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}_test exp/make_mfcc $mfccdir
+      utils/fix_data_dir.sh data/fame_${task}_${subtask}_${sets}_test
+      steps/make_mfcc.sh --mfcc-config conf/mfcc_hires_16k.conf --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}_test_dnn exp/make_mfcc $mfccdir
+      utils/fix_data_dir.sh data/fame_${task}_${subtask}_${sets}_test_dnn
+
+    done
+  done
+done
+
+for task in ageing; do
+  for subtask in 3sec 10sec 30sec; do
+    for sets in eval; do
+      for year in _1t3 _4t10 _mt10; do
+
+        echo "Extracting MFCC features for data/fame_${task}_${subtask}_${sets}${year}.."
+        steps/make_mfcc.sh --mfcc-config conf/mfcc_16k.conf --nj 100 --cmd "$train_cmd" \
+            data/fame_${task}_${subtask}_${sets}${year}_enroll exp/make_mfcc $mfccdir
+        utils/fix_data_dir.sh data/fame_${task}_${subtask}_${sets}${year}_enroll
+        steps/make_mfcc.sh --mfcc-config conf/mfcc_hires_16k.conf --nj 100 --cmd "$train_cmd" \
+            data/fame_${task}_${subtask}_${sets}${year}_enroll_dnn exp/make_mfcc $mfccdir
+        utils/fix_data_dir.sh data/fame_${task}_${subtask}_${sets}${year}_enroll_dnn
+
+        steps/make_mfcc.sh --mfcc-config conf/mfcc_16k.conf --nj 100 --cmd "$train_cmd" \
+            data/fame_${task}_${subtask}_${sets}${year}_test exp/make_mfcc $mfccdir
+        utils/fix_data_dir.sh data/fame_${task}_${subtask}_${sets}${year}_test
+        steps/make_mfcc.sh --mfcc-config conf/mfcc_hires_16k.conf --nj 100 --cmd "$train_cmd" \
+            data/fame_${task}_${subtask}_${sets}${year}_test_dnn exp/make_mfcc $mfccdir
+        utils/fix_data_dir.sh data/fame_${task}_${subtask}_${sets}${year}_test_dnn
+
+      done
+    done
+  done
+done
+
+# VAD computation
+
+echo "Computing VAD for data/train.."
+
+sid/compute_vad_decision.sh --nj 100 --cmd "$train_cmd" \
+    data/train exp/make_vad $vaddir
+
+for task in complete ageing; do
+  for subtask in 3sec 10sec 30sec; do
+    for sets in eval; do
+
+      echo "Computing VAD for data/fame_${task}_${subtask}_${sets}.."
+      sid/compute_vad_decision.sh --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}_enroll exp/make_vad $vaddir
+      sid/compute_vad_decision.sh --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}_test exp/make_vad $vaddir 
+
+    done
+  done
+done
+
+for task in ageing; do
+  for subtask in 3sec 10sec 30sec; do
+    for sets in eval; do
+      for year in _1t3 _4t10 _mt10; do
+
+      echo "Computing VAD for data/fame_${task}_${subtask}_${sets}${year}.."
+      sid/compute_vad_decision.sh --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}${year}_enroll exp/make_vad $vaddir
+      sid/compute_vad_decision.sh --nj 100 --cmd "$train_cmd" \
+          data/fame_${task}_${subtask}_${sets}${year}_test exp/make_vad $vaddir
```

Comment:
Indent, lines 190-194.

Reply:
Fixed

Reply:
I think you forgot to push some changes. Doesn't look fixed.